app/javascript/controllers/select2_controller.js

```diff
@@ -5,17 +5,29 @@ import "select2"
 export default class extends Controller {
   static values = {
     config: { type: Object, default: {} },
-  }
+    hideDropdown: { type: Boolean, default: false }
```

Comment:
Changes in this file are:
* Allow for more than one select2 on the same page (this was previously broken)
* Add a new parameter that can hide the dropdown (for the 'tags' use-case)